source/features/unfinished-comments.tsx

```diff
@@ -9,7 +9,7 @@ function hasDraftComments(): boolean {
 	// `[disabled]` excludes the PR description field that `wait-for-build` disables while it waits
 	return select.all<HTMLTextAreaElement>('textarea:not([disabled])').some(textarea =>
 		textarea.value !== textarea.textContent && // Exclude comments being edited but not yet changed (and empty comment fields)
-		textarea.offsetWidth > 0 && // Exclude invisible fields
+		(textarea.offsetWidth > 0 || select('.js-preview-body', textarea.form!)!.textContent !== 'Nothing to preview') && // Exclude invisible fields that aren't being previewed
```

Comment:
Maybe this should be
```suggestion
		(textarea.offsetWidth > 0 || select('.js-preview-body', textarea.form!)?.textContent !== 'Nothing to preview') && // Exclude invisible fields that aren't being previewed
```
?

Reply:
I think so because `textarea` isn't necessarily a comment.

However I think we should drop this check altogether since there are too many exceptions. For examples when you type a final review comment (in the "Review changes" dropdown) and then you close it.

Can you find a list all situations where there are hidden textareas that should be ignored?

Reply:
After a bit of testing, I agree we can drop this check altogether. The only reason it was introduced in the first place is because I relied on just `textarea.value.length` to check for draft comments. But afaik the improved condition we're using now makes checking for hidden textareas redundant.

> For examples when you type a final review comment (in the "Review changes" dropdown) and then you close it.

There's actually a case where it doesn't work regardless: when you review your own PR. It's caused by the disabled `Approve` and `Request change` buttons. I don't know how common that situation is though.

> Can you find a list all situations where there are hidden textareas that should be ignored?

After testing the most common cases I haven't found any. I'll try to think of possible edge cases.